applications/nrf5340_audio/src/audio/sw_codec_select.c

```diff
@@ -21,8 +21,8 @@ LOG_MODULE_REGISTER(sw_codec_select, CONFIG_SW_CODEC_SELECT_LOG_LEVEL);
 
 static struct sw_codec_config m_config;
 
-static struct sample_rate_converter_ctx encoder_converters[AUDIO_CH_NUM];
-static struct sample_rate_converter_ctx decoder_converters[AUDIO_CH_NUM];
+static struct sample_rate_converter_ctx encoder_converters[CONFIG_AUDIO_ENCODE_CHANNELS_MAX];
```

Comment:
Almost all changes in sw_codec_select in this PR is made to make it run. I suspect that most of this will be changed in to N channels in OCT-3398. Discuss.